clients/src/main/java/org/apache/kafka/clients/admin/ListTransactionsOptions.java

```diff
@@ -35,6 +35,7 @@ public class ListTransactionsOptions extends AbstractOptions<ListTransactionsOpt
     private Set<TransactionState> filteredStates = Collections.emptySet();
     private Set<Long> filteredProducerIds = Collections.emptySet();
 
+    private Long durationFilter = 0L;
```

Comment:
nit: any reason we chose default to be zero and not -1?

Reply:
Updated default value to -1L in [81173c8](https://github.com/apache/kafka/pull/15384/commits/81173c89dbf50a98671694f99a051dad1e0b1821)